test/tools/libtesteth/TestSuite.cpp

```diff
@@ -150,7 +150,7 @@ void TestSuite::runAllTestsInFolder(string const& _testFolder) const
 	for (auto const& file: files)
 	{
 		testOutput.showProgress();
-		testOutput.setCurrentTestFileName(file.filename().string());
+		testOutput.setCurrentTestFileName(file);
```

Comment:
This change is not related to other changes in the same commit.

Reply:
is there a way to remove a particular change from a commit?

Reply:
changed

Reply:
Why do you use full path now? The name of the function is confusing.

Reply:
because file is not could be Filler.json and Filler.yml 
boost path provide interface to get filename and extention

Reply:
So you should rename the function to `setCurrentTestFile()`.